config/routes.rb

```diff
@@ -6,9 +6,17 @@
 
   resources :episodes do
     resources :announcements
+    resources :timecodes, only: [:create]
   end
 
   get '/admin', to: 'dashboard#index'
   post '/admin/telegram', to: 'dashboard#telegram'
+
   get '*path', to: redirect('/')
+
+  post '/update_youtube_info/:id', to: 'episodes#update_youtube_info'
```

Comment:
You have versions of youtube_info?

Reply:
You know, we don't need this route at all anymore :) This action was moved to callback.